bitnami/common/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
 type: library
-version: 2.13.2
```

Comment:
to have a backward compatible change, you need to keep the function with the typo, so both "common.capabilities.admisionConfiguration.supported" (adding a comment that it's deprecated) and "common.capabilities.admissionConfiguration.supported"

current change should bump to 3.0.0

ref: https://semver.org/

Reply:
Absolutely spot on.

Any objections to use the depreciated approach?

Reply:
You're right @jouve but we were quiet lucky this time and the macro hasn't been adopted by any chart in the catalog yet, see:
```bash
grep -Ri "common.capabilities.admisionConfiguration.supported" .
```
That's why I'd do an exemption this time.

Reply:
@juan131 @jouve

1. Can I consider this as resolved?
2. Do you need anything else from my side?

Reply:
Yes, let's consider it solved